objects/oplayercontroller/Step_0.gml

```diff
@@ -842,12 +893,47 @@ switch state
 	{
 		if (grounded)
 		{	
-			GroundedAttackScript(selectedCharacter.NeutralSpecial, true, selectedCharacter.NeutralSpecial.AirMovementData.GravityScale, selectedCharacter.NeutralSpecial.AirMovementData.FallScale, true, true);	
+			GroundedAttackScript(selectedCharacter.NeutralSpecial, true, selectedCharacter.NeutralSpecial.AirMovementData.GravityScale, selectedCharacter.NeutralSpecial.AirMovementData.FallScale, true, true);
 		} 
 		else 
 		{
 			JumpingAttackScript(selectedCharacter.NeutralSpecial, false, selectedCharacter.NeutralSpecial.AirMovementData.GravityScale, selectedCharacter.NeutralSpecial.AirMovementData.FallScale);
 		}
+		
+		// NOTE FOR DESIGNERS:
+		// In the editor, the following additions will be made to each special move to allow motion input editing:
+		// - The number of inputs (int)
+		//   - Each input in numpad notation (int)
+		//   - Input Window Start (int)
+		//   - Input Window End (int)
+		//   - Does the move change as soon as the player performs the input? (bool)
+		//     - If not, set the frame when the move changes (int)
+		//   - Which move will be performed (data type unknown for now)
+		//     - Starting animation frame (int)
+		//     - Is the move cancellable? (for things like rekkas) (bool)
+		
+		motionInput[0] = 236;
+		motionInput[1] = 41236;
+		SetMotionInputs(motionInput, array_length(motionInput), 1, 27, 999, true);
+		
+		// Checks to see if the special move can be changed
+		if (CheckChange())
+		{
+			if (enhanced[0])
+			{
+				CancelIntoMove(eState.NEUTRAL_SPECIAL, selectedCharacter.NeutralSpecial.SpriteId, 1);

```

Comment:
Why are you cancelling into neutral special? CancelIntoMove should have the move you want to cancel into, not what you are canceling out of. I tested it by setting it to crouching medium and it worked fine, so I'd like some clarification here.

Reply:
There's something about the CancelIntoMove script that allows the follow-ups to connect. Through my testing, the follow-ups wouldn't connect without it. I'll look further into how to make them connect without it.

Reply:
Actually, I think it would be fine if I kept the CancelIntoMove and changed it to the follow-up move. It worked when I put the specials in that script, so I had it that way before.